qcodes/instrument_drivers/Minicircuits/USB_SPDT.py

```diff
@@ -0,0 +1,88 @@
+from qcodes import Instrument
+from qcodes.utils import validators as vals
+from qcodes.instrument.channel import InstrumentChannel, ChannelList
+
```

Comment:
we need to import the clr (Pythonnet). Would be good to keep the import error message also.

Reply:
that is a good point, I totally forgot the import.